tasks/cephfs/test_volume_client.py

```diff
@@ -196,7 +206,9 @@ def test_idempotency(self):
         guest_entity = "guest"
         group_id = "grpid"
         volume_id = "volid"
-        self._volume_client_python(self.mount_b, dedent("""
+        self._volume_client_python(self.mount_b,
+            volume_prefix,
```

Comment:
volume_prefix not defined here, or later in test_data_isolated (it was defined in test_lifecycle)